source/excerptexport/views.py

```diff
@@ -36,10 +37,13 @@ def new_excerpt_export(request):
 
 @login_required(login_url='/admin/')
 def create_excerpt_export(request):
-    viewContext = {}
```

Comment:
Maybe it would improve the readability and traceability where the variable is created?